metron-interface/metron-rest/README.md

```diff
@@ -0,0 +1,458 @@
+| [ `DELETE /api/v1/sensor/indexing/config/{name}`](#delete-apiv1sensorindexingconfig{name})|
+| [ `POST /api/v1/sensor/indexing/config/{name}`](#post-apiv1sensorindexingconfig{name})|
+| [ `GET /api/v1/sensor/indexing/config/{name}`](#get-apiv1sensorindexingconfig{name})|
+| [ `POST /api/v1/sensor/parser/config`](#post-apiv1sensorparserconfig)|
+| [ `GET /api/v1/sensor/parser/config`](#get-apiv1sensorparserconfig)|
+| [ `GET /api/v1/sensor/parser/config/history`](#get-apiv1sensorparserconfighistory)|
+| [ `GET /api/v1/sensor/parser/config/history/history/{name}`](#get-apiv1sensorparserconfighistoryhistory{name})|
+| [ `GET /api/v1/sensor/parser/config/history/{name}`](#get-apiv1sensorparserconfighistory{name})|
+| [ `GET /api/v1/sensor/parser/config/list/available`](#get-apiv1sensorparserconfiglistavailable)|
+| [ `POST /api/v1/sensor/parser/config/parseMessage`](#post-apiv1sensorparserconfigparsemessage)|
+| [ `GET /api/v1/sensor/parser/config/reload/available`](#get-apiv1sensorparserconfigreloadavailable)|
+| [ `DELETE /api/v1/sensor/parser/config/{name}`](#delete-apiv1sensorparserconfig{name})|
+| [ `GET /api/v1/sensor/parser/config/{name}`](#get-apiv1sensorparserconfig{name})|
+| [ `GET /api/v1/storm`](#get-apiv1storm)|
+| [ `GET /api/v1/storm/client/status`](#get-apiv1stormclientstatus)|
+| [ `GET /api/v1/storm/enrichment`](#get-apiv1stormenrichment)|
+| [ `GET /api/v1/storm/enrichment/activate`](#get-apiv1stormenrichmentactivate)|
+| [ `GET /api/v1/storm/enrichment/deactivate`](#get-apiv1stormenrichmentdeactivate)|
+| [ `GET /api/v1/storm/enrichment/start`](#get-apiv1stormenrichmentstart)|
+| [ `GET /api/v1/storm/enrichment/stop`](#get-apiv1stormenrichmentstop)|
+| [ `GET /api/v1/storm/indexing`](#get-apiv1stormindexing)|
+| [ `GET /api/v1/storm/indexing/activate`](#get-apiv1stormindexingactivate)|
+| [ `GET /api/v1/storm/indexing/deactivate`](#get-apiv1stormindexingdeactivate)|
+| [ `GET /api/v1/storm/indexing/start`](#get-apiv1stormindexingstart)|
+| [ `GET /api/v1/storm/indexing/stop`](#get-apiv1stormindexingstop)|
+| [ `GET /api/v1/storm/parser/activate/{name}`](#get-apiv1stormparseractivate{name})|
+| [ `GET /api/v1/storm/parser/deactivate/{name}`](#get-apiv1stormparserdeactivate{name})|
+| [ `GET /api/v1/storm/parser/start/{name}`](#get-apiv1stormparserstart{name})|
+| [ `GET /api/v1/storm/parser/stop/{name}`](#get-apiv1stormparserstop{name})|
+| [ `GET /api/v1/storm/{name}`](#get-apiv1storm{name})|
+| [ `GET /api/v1/transformation/list`](#get-apiv1transformationlist)|
+| [ `GET /api/v1/transformation/list/functions`](#get-apiv1transformationlistfunctions)|
+| [ `GET /api/v1/transformation/list/simple/functions`](#get-apiv1transformationlistsimplefunctions)|
+| [ `POST /api/v1/transformation/validate`](#post-apiv1transformationvalidate)|
+| [ `POST /api/v1/transformation/validate/rules`](#post-apiv1transformationvalidaterules)|
+| [ `GET /api/v1/user`](#get-apiv1user)|
+
+### `GET /api/v1/global/config`
```

Comment:
There is a utility included (ReadMeUtils) that generates the README from the swagger so I would consider it the source of truth.  I like the idea of putting Swagger into our general docs though.